cli/init.md

```diff
@@ -81,84 +86,98 @@ What happens behind the scenes?
     |_ #current-cloud-backend/
         |_ amplify-meta.json
     |_ .config
-        |_ aws-info.json
+        |_ local-aws-info.json
+        |_ local-env-info.json
         |_ project-config.json
     |_ backend/
         |_amplify-meta.json
-    |_.amplifyrc
+    |_team-provider-info.json
 ```
 
-- The `amplify/backend` directory contains all the local changes to your backend configurations--such as the CloudFormation templates--when you add resources to your project using the `amplify add <category>` command. It also contains the `amplify-meta.json` file which contains all the metadata tied to your resources which you add via the Amplify CLI. This file has the following format after you first initialize your project:
+- The `amplify/backend` directory contains all the local changes to your backend configurations--such as the CloudFormation templates--when you add resources to your project using the `amplify add <category>` command. It contains the `backend-config.json` file which is a structure to represent your backend infrastructure and certain CLI runtime files like the `amplify-meta.json` file which contains all the metadata tied to your resources which you add via the Amplify CLI.
 
-```
-{
-"providers": {
-    "awscloudformation": {
-        "AuthRoleName": "test890-xxxxxxx-authRole",
-        "UnauthRoleArn": "arn:aws:iam::132393967379:role/test890-xxxxx-unauthRole",
-        "AuthRoleArn": "arn:aws:iam::132393967379:role/test890-xxxxxxxx-authRole",
-        "Region": "us-east-1",
-        "DeploymentBucketName": "test890-xxxxxxx-deployment",
-        "UnauthRoleName": "test890-xxxxxx-unauthRole",
-        "StackName": "test890-xxxxxxxxx",
-        "StackId": "arn:aws:cloudformation:us-east-1:132393967379:stack/test890-20181006145850/fe8b16d0-xxxxxxxxxxx"
-    }
-}
-}
-```
 
-- The `#current-cloud-backend` directory has exactly a similar structure to the `backend` directory. The only differences between it and the `backend` directory are that it has the configurations that reflect what resources were deployed in the cloud with your last `amplify push` command and it helps the CLI diff between the configuration of the resources already provisioned in the cloud and what is currently in your local `backend` directory (which reflects your local changes).
+- The `#current-cloud-backend` directory has a similar structure to the `backend` directory. The only differences between it and the `backend` directory are that it has the configurations that reflect what resources were deployed in the cloud with your last `amplify push` command and it helps the CLI diff between the configuration of the resources already provisioned in the cloud and what is currently in your local `backend` directory (which reflects your local changes).
 
-- The `.config` directory consists of the metadata files tied to your project. The `project-config.json` file has the following format:
+- The `.config` directory consists of the metadata files tied to your project. 
```

Comment:
Suggested rewording --> The `project-config.json` file, which can be safely checked into a version control system,

Reply:
Suggested rewording --> The `project-config.json` file, which can be safely checked into a version control system, represents information specific to the app you're building. For e.g. a sample format shown below reflects the framework you're using for your app: